src/Airy/cairy.jl

```diff
@@ -263,29 +248,15 @@ function _airybiprime(z::Complex{T}) where T <: Union{Float32, Float64}
         end
     end
     x, y = real(z), imag(z)
-    airy_large_argument_cutoff(z) && return airybi_large_args(z)[2]
-    airybi_power_series_cutoff(x, y) && return airybi_power_series(z)[2]
-
-    if x > zero(T)
-        zz = T(2/3) * z * sqrt(z)
-        shift = 20
-        order = T(2/3)
-        inu, inum1 = besseli_power_series_inu_inum1(order + shift, zz)
-        inu, inum1 = besselk_down_recurrence(zz, inum1, inu, order + shift - 1, order)
-
-        inu2, inum2 = besseli_power_series_inu_inum1(-order + shift, zz)
-        inu2, inum2 = besselk_down_recurrence(zz, inum2, inu2, -order + shift - 1, -order)
-        return z / sqrt(3) * (inu + inu2)
+
+    if airy_large_argument_cutoff(z)
+        return airybi_large_args(z)[2]
+    elseif x > 1.1 && y > 4.2
+        return airybiprime_levin(z, Val(16))
+    elseif angle(z) < 7pi/8 || x > -4.5
+        return airybi_power_series(z)[2]
     else
-        if iszero(y)
-            xabs = abs(x)
-            xx = T(2/3) * xabs * sqrt(xabs)
-            Jv, Yv = besseljy_positive_args(T(2)/3, xx)
-            Jmv = -(Jv + sqrt(T(3))*Yv) / 2
-            return convert(eltype(z), xabs * (Jv + Jmv) / sqrt(T(3)))
-        else
-            return -(cispi(T(2/3)) * _airybiprime(-z*cispi(one(T)/3)) + cispi(-T(2/3)) * _airybiprime(-z*cispi(-one(T)/3)))
-        end
+        return cispi(-5/6) * airyaiprime(z * cispi(-2/3)) + cispi(5/6) * airyaiprime(z*cispi(2/3))
```

Comment:
this will promote for `Float32`

Reply:
Ya will work on separate `Float32` implementations as they should never have to hit that branch.